pkg/providers/v1/aws.go

```diff
@@ -249,6 +249,9 @@ const volumeAttachmentStuck = "VolumeAttachmentStuck"
 // Indicates that a node has volumes stuck in attaching state and hence it is not fit for scheduling more pods
 const nodeWithImpairedVolumes = "NodeWithImpairedVolumes"
 
+const sourceKey = "x-amz-source-arn"
+const accountKey = "x-amz-source-account"
```

Comment:
nit: some clearer names for these constants
```suggestion
const headerSourceArn = "x-amz-source-arn"
const headerSourceAccount = "x-amz-source-account"
```